src/chains/models.py

```diff
@@ -10,6 +17,12 @@ class Chain(models.Model):
     currency_symbol = models.CharField(max_length=255)
     currency_decimals = models.IntegerField(default=18)
     transaction_service_url = models.URLField(null=True)
+    theme_text_color = models.CharField(
+        validators=[color_validator], max_length=9, default="#fff"
+    )
+    theme_background_color = models.CharField(
+        validators=[color_validator], max_length=9, default="#000"
+    )
```

Comment:
@Uxio0 I thought about making a custom field for the color and I was reading more about it. I think for the use case that we have right now we can have it as a `CharField` with a `RegexValidator` what do you think?

Reply:
I think text with validator will work perfect. I would define a new field if you see yourself using in many places. There are examples of custom fields in `gnosis-py`

Reply:
Sounds good 👍 If we end up using this in more places then I will consider adding a new field